docs/source/documentation/javascript/_event-bus.md

```diff
@@ -80,16 +80,16 @@ Many of the filter dropdown channels emit and listen for the following options o
 | filterDropdown:active:clear | Clear all active items from the dropdown | Options Object | `clearActive()` |
 | filterDropdown:toggle | Toggle a dropdown open and closed | Options Object | `toggleDropdown()` |
 | filterDropdown:open | Open a specific dropdown | Options Object | `openList()` |
-| filterDropdown:close | Closes all dropdowns | n/a. | `closeList()` |
+| filterDropdown:close | Closes all dropdowns | n/a | `closeList()` |
 | keyboard:escape | Closes all dropdowns | n/a | `closeList()` |
 
 #### Modal
 
 | Event | Description | Emits | Function |
 | --- | --- | --- | --- |
 | modal:bind | Binds dom to event listeners | n/a | `bindModals()` |
-| modal:open | Opens a specific modal | `{id: modalId }` |  `openModal()` |
-| modal:close | Closes all modals | n/a | `closeModal()` |
+| modal:open | Opens a modal specified by data-modal attribute | `{id: modalid}` |  `openModal()` |
+| modal:close | Closes all modals or closes a modal specified by data-modal attribute | n/a; `{id: modalid}` | `closeModal()` |
```

Comment:
I think n/a is actually more correct here?

Reply:
Yep in fact there will only be one open at a time anyway.